### PR TITLE
Add Loïc Mathieu (myself)

### PR DESCRIPTION
**Pull request by loicmathieu**

### CODE REVIEW

File: javapeople.yaml [hunk 1]
```diff
@@ -687,6 +687,12 @@
   twitter: "lbkulinski"
   fediverse: "lbkulinski@mastodon.world"
   github: "lbkulinski"
+- name: "Loïc Mathieu"
+  twitter: "loicmathieu"
+  fediverse: "@loicmathieu@mastodon.online"
```
Comment (loicmathieu) [inline javapeople.yaml:692]:
```suggestion
  fediverse: "loicmathieu@mastodon.online"
```

File: javapeople.yaml [hunk 2]
```diff
@@ -687,6 +687,12 @@
...
+  bluesky: "@loicmathieu.fr"
```
Comment (loicmathieu) [inline javapeople.yaml:693]:
```suggestion
  bluesky: "loicmathieu.fr"
```

File: javapeople.yaml [hunk 3]
```diff
@@ -687,6 +687,12 @@
   twitter: "lbkulinski"
   fediverse: "lbkulinski@mastodon.world"
   github: "lbkulinski"
+- name: "Loïc Mathieu"
+  twitter: "loicmathieu"
+  fediverse: "loicmathieu@mastodon.online"
+  bluesky: "loicmathieu.fr"
+  github: "loicmathieu"
+  linkedin: "loïc-mathieu-475b144"
```
Comment (scordio) [inline javapeople.yaml:695]: The LinkedIn handle pattern appears to be not flexible enough in this case 🙂 

https://github.com/marchof/javabubble/blob/e3dc84e0892e1f95a706f742445f8d388cc15a0a/generator/src/main/java/org/javabubble/generator/model/LinkedInHandle.java#L7

According to [the docs](https://www.linkedin.com/help/linkedin/answer/a542685/manage-your-public-profile-url):

> Your custom URL can be 3-100 characters long and should not contain spaces, symbols, special characters, or the word LinkedIn.

We might align the implementation to such rules.

Comment (loicmathieu) [reply javapeople.yaml:695]: If it's an issue, you can remove my LinkedIn account

Comment (marchof) [reply javapeople.yaml:695]: No, we fix it!

Comment (scordio) [reply javapeople.yaml:695]: I switched to the word class with Unicode support, which is good enough for umlaut characters.

**Review by marchof — APPROVED** [review]

**Pull request merged**